neotest/main.d

```diff
@@ -108,6 +108,8 @@ class Fill : DhtTest
             this.channels = ["test".dup];
         else
             this.channels = channels;
+
+        Stdout.formatln("Fill: {} records, channels: {}", this.max, this.channels);
```

Comment:
This doesn't add printout _after_ the command, but when it's constructed? (regarding the commit message)

Reply:
:+1: removed this commit. (It's not necessary.)